docs/source/crawler_guide.rst

```diff
@@ -54,9 +54,9 @@ Start Button
 
 Stop Button
 -----------
-    Symbolized by the "stop" button. Stops the crawl.
+   Symbolized by the "stop" button. Stops the crawl.
 
-    In the case of Ache, the crawler stops immediately. In the case of Nutch, the crawler stops after it has finished the current round. This is in order to prevent data corruption that can occur when killing the Nutch process.
+   In the case of Ache, the crawler stops immediately. In the case of Nutch, the crawler stops after it has finished the current round. This is in order to prevent data corruption that can occur when killing the Nutch process.
```

Comment:
No longer true.  Now it stops after the current task.  All data from the current crawl round is lost though.